rpm_spec/gui-common.spec.in

```diff
@@ -21,35 +21,32 @@
 #
 #
 
-%{!?version: %define version %(cat version)}
-
-
 Name:		qubes-gui-common-devel
-Version:	%{version}
+Version:	@VERSION@
 Release:	1%{?dist}
 Summary:	Common files for Qubes GUI - protocol headers
 
 Group:		Qubes
 License:	GPL
 URL:		http://www.qubes-os.org
 
-%define _builddir %(pwd)
+Source0: %{name}-%{version}.tar.gz
 
 %description
 Protocol description headers for Qubes GUI
 
 %prep
 
-%build
+%setup -q
 
 %install
 mkdir -p %{buildroot}/usr/include
-cp include/*.h %{buildroot}/usr/include/
+cp $(PWD)include/*.h %{buildroot}/usr/include/
```

Comment:
This is shell not make, so it looks for (non-existing) `PWD` command. https://travis-ci.org/QubesOS/qubes-gui-common/jobs/361809885#L2687

Also, this is unnecessary, relative paths are handled in relation to current directory.